src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorBackgroundLinkHighlighter.java

```diff
@@ -147,7 +153,22 @@ public void execute()
          {
             TextFileType fileType = editor_.getFileType();
             highlighters_.clear();
-            highlighters_.add(webLinkHighlighter());
+            if (pUserPrefs_.get().highlightWebLink().getValue())
+	    {
```

Comment:
Nit: the indentation's off here by a couple of spaces (brace is less indented than if)

Reply:
Addressed in 70cd3b7.